### ISSUE TITLE
Disable Debug level logs from com.epam.reportportal on console

### ISSUE THREAD

**Issue by PavelGermanovich** [reportportal/reportportal#737]
hello, i have integrated reportportal with my testing framework (java+ testng + log4j)
Everything looks fine except one issue: i specified log level as INFO in log4j properties, but still DEBUG logs from com.epam.reportportal are still displayed on console, such as:

> [rp-io-0] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Launch file create successfully completed

Log4j properties file is the following:
`log4j.rootLogger=INFO, stdout, reportportal

log4j.appender.stdout=org.apache.log4j.ConsoleAppender
log4j.appender.stdout.encoding=UTF-8

#Cp1251
#log4j.appender.stdout.encoding=Cp1251
log4j.appender.stdout.target=System.out
log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
log4j.appender.stdout.layout.conversionPattern=%d{ABSOLUTE} %5p - %m%n

log4j.appender.reportportal=com.epam.ta.reportportal.log4j.appender.ReportPortalAppender
log4j.appender.reportportal.layout=org.apache.log4j.PatternLayout
log4j.appender.reportportal.layout.ConversionPattern=%d{ABSOLUTE} %5p - %m%n
`

Those debug messages are not sent to the reportportal itself, but displayed in console, and i want to get rid of them, could you please provide any ideas how to do this? 
thanks in advance

**Comment by evjlobanova** [comment]
@PavelGermanovich which version of reportportal and agents do you use?

**Comment by PavelGermanovich** [comment]
@evjlobanova 

```
            <groupId>com.epam.reportportal</groupId>
            <artifactId>agent-java-testng</artifactId>
            <version>4.2.1</version>
```
```
           <groupId>com.epam.reportportal</groupId>
            <artifactId>logger-java-log4j</artifactId>
            <version>4.0.1</version>
```
```
   <artifactId>log4j-api</artifactId>
            <version>2.10.0</version>
```
```
            <artifactId>log4j-core</artifactId>
            <version>2.10.0</version>
```

**Comment by HardNorth** [comment]
@PavelGermanovich Well, RP uses slf4j to log its data. It looks like in your case that library just wasn't configure and picked up standard console logger. I would recommend to include slf4j-log4j bridge to route these logs to your logger:
```        <dependency>
            <groupId>org.apache.logging.log4j</groupId>
            <artifactId>log4j-slf4j-impl</artifactId>
            <version>2.12.1</version>
        </dependency>```

**Comment by HardNorth** [comment]
Closed due to two weeks of inactivity

**Comment by PavelGermanovich** [comment]
The problem was reproduced because two log4j.properties files were exist in the project (one in java -> resources and the other in test -> resources). I removed the redundant second file and it works now, thanks

**Comment by VolhaKarenko** [comment]
The issue is fixed in version 5.7
